Base/usr/share/man/man2/seteuid.md

```diff
@@ -0,0 +1,40 @@
+## Name
+
+seteuid, setegid - set effective user / group id
+
+## Synopsis
+
+```**c++
+#include <unistd.h>
+
+int seteuid(uid_t);
+int setegid(gid_t);
+```
+
+## Description
+
+Sets the effective user or group id.
+
+The effective user or group id is the id used to decide the permissions of the current process. An effective user id of 0 determines if the user is superuser.
```

Comment:
`#include <english-ain't-my-first-language.h>`

"Decide permissions" sounds weird to me; perhaps "decide on permissions" or "determine permissions"?

Besides, such an explanation about how these IDs work should be in `setuid`(7), not in (2).

Reply:
Ok, removed this line.